src/main/java/step2/view/ResultView.java

```diff
@@ -0,0 +1,36 @@
+package step2.view;
+
+import step2.domain.*;
+
+import java.util.List;
+
+public class ResultView {
+    private final static String PERSON_NAME_FORMAT = "%6s";
+    private final static String NEW_LINE = System.lineSeparator();
+
+    public static void print(Persons persons, Ladder ladder) {
+        arrangePersons(persons.getPersons());
+        System.out.print(NEW_LINE);
+        arrangeLadder(ladder);
+    }
+
+    private static void arrangePersons(List<Person> persons){
+       persons.stream()
+              .map(person -> formatName(person.getName()))
+              .forEach(name -> {
+                  System.out.print(name);
+              });
```

Comment:
람다 내부에서 호출하고 있는 메소드와 파라미터의 갯수와 타입이 동일해서 아래처럼 메소드 레퍼런스로 변경이 가능해요.
```suggestion
              .forEach(System.out::print);
```